doc/source/examples.rst

```diff
@@ -57,20 +57,6 @@ Here we visualize a neuronal morphology:
     >>> fig, ax = viewer.draw(nrn, mode='3d') # valid modes '2d', '3d', 'dendrogram'
     >>> fig.show()
 
-Basic feature extraction example
```

Comment:
It's in the spirit of getting people to use morph_stats instead of rolling their own.